### PR TITLE
Two documentation updates

### PR DESCRIPTION
**Pull request by keszybz**
The first patch should be backported.

### CODE REVIEW

**Review by bluca — APPROVED** [review]

**Comment by keszybz** [comment]
I put up both patches for v254.1, I think the advice change in the second one is appropriate for stable too.

File: man/systemd.service.xml [hunk 1]
```diff
@@ -155,60 +155,72 @@
...
-            <listitem><para>The <option>exec</option> type is similar to <option>simple</option>, but the service
-            manager will consider the unit started immediately after the main service binary has been executed. The service
-            manager will delay starting of follow-up units until that point. (Or in other words:
-            <option>simple</option> proceeds with further jobs right after <function>fork()</function> returns, while
-            <option>exec</option> will not proceed before both <function>fork()</function> and
-            <function>execve()</function> in the service process succeeded.) Note that this means <command>systemctl
-            start</command> command lines for <option>exec</option> services will report failure when the service's
-            binary cannot be invoked successfully (for example because the selected <varname>User=</varname> doesn't
+            will consider the unit started immediately after the main service process has been forked off.
+            <emphasis>The use of this type is discouraged, use <option>exec</option> instead.
```
Comment (poettering) [inline man/systemd.service.xml:167]: uh, i am not on board with this. Type=simple is a ton faster that Type=exec. I am not sure whether Type=exec is really the better choice for all cases. It just means we have to wait a lot when spawning services.

Type=exec makes us block on NSS lookups and DM/udev and such, i.e. operations on other subsytems via IPC. It should not be used willy-nilly. 

Yeah, type=exec has benefits, but it comes at a non-trivial cost in deadlock chances and well, speed. so i am not sure it's so clear cut that Type=simple is discouraged and type=exec is the way to got. I really depends on the usecase.

File: man/systemd.service.xml [hunk 2]
```diff
@@ -249,27 +268,25 @@
             anyway.</para></listitem>
           </itemizedlist>
 
-          <para>It is generally recommended to use <varname>Type=</varname><option>simple</option> for
+          <para>It is generally recommended to use <varname>Type=</varname><option>exec</option> for
           long-running services whenever possible, as it is the simplest and fastest option. However, as this
```
Comment (poettering) [inline man/systemd.service.xml:272]: uh, this is simply wrong. it's definitely *not* the "fastest" nor "simplest" option.

**Comment by poettering** [comment]
I toned this down a bit, and corrected somethings: → #28745

**Pull request merged**